tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+envlist = py27, lint, coverage-report
+skip_missing_interpreters = true
+
+[testenv]
+deps = -rrequirements.txt
+commands = pytest \
+  --cov=girder_worker \
+  --cov-config=.coveragerc.pytest \
+  # Surpress printing coverage, defer this to testenv:coverage-report
+  --cov-report= \
+  {posargs}
+
+# This report is used to merge coverage reports from testenv and from
+# the legacy tests run by ctest. When just running 'tox' it will print
+# the pytest coverage report. When running ctest && tox it will print
+# the combine coverage.
+[testenv:coverage-report]
```

Comment:
This environment can't be run twice in a row (coverage combine fails after it's run once).

Reply:
is there a way to depend on other tests being run first with tox?